### ISSUE TITLE
Can't use arrow keys in output window

### ISSUE THREAD

**Issue by camilstaps** [lervag/vimtex#968]
### Explain the issue

In the output window, pressing an arrow key does not move the cursor (what it usually does), but closes the output window, starts insert mode in the then focused window and enters an `A` (up key), `B` (down key), `C` (left key) or `D` (right key).

1. Steps to reproduce: the below `.vimrc`, opening any TeX document, calling `:VimtexCompile` and `:VimtexCompileOutput` and then trying to move around with the arrow keys.
2. Expected output: the arrow keys move the cursor, like normally.
3. Observed output: the output window is closed, insert mode is started in the then focused window and a character is added.

### Minimal working example

```tex
\documentclass{minimal}
\begin{document}
Hello World!
\end{document}
```

### Minimal vimrc file

```vim
set nocompatible
let &rtp  = '~/.vim/bundle/vimtex,' . &rtp
let &rtp .= ',~/.vim/bundle/vimtex/after'
filetype plugin indent on
syntax enable
```

**Comment by lervag** [comment]
The reason for this is that in terminal Vim the arrowkeys are "composite" keys that use escape codes. E.g., arrow left sends `^]OD`, where `^]` is the escape character. The keys may differ between terminals/systems. And I've mapped escape to close the window. Obv., I can disable that mapping since `q` already does the same.

Note: neovim handles this in a better way, so this problem only occurs in regular Vim. Also, `gvim` handles this as expected.

I'm pushing an update where the `<esc>` is not mapped in regular Vim in terminal, which solves your problem. However, if anyone finds this annoying, I am tempted to revert the update and instead argue that you should learn to avoid the arrow keys. :)

**Comment by camilstaps** [comment]
Excellent, thanks for the speedy reply. I know, I should learn to avoid the arrow keys :) thanks nevertheless.

The commit accidentally closed #958 again, so I'm closing this manually.

**Comment by lervag** [comment]
Thanks, it seems I was a little _too_ speedy. :)
